Ming-omni/test_infer.py

```diff
@@ -0,0 +1,111 @@
+import os
+import torch
+from transformers import AutoProcessor
+
+from modeling_bailingmm import BailingMMNativeForConditionalGeneration
+
+
+if __name__ == '__main__':
+    processor = AutoProcessor.from_pretrained(".", trust_remote_code=True)
+    model_path = "MODEL_PATH"
+    model = BailingMMNativeForConditionalGeneration.from_pretrained(
+        model_path,
+        torch_dtype=torch.bfloat16,
+        low_cpu_mem_usage=True,
+        attn_implementation="flash_attention_2",
+    ).to("cuda")
+
+    vision_path = "/input/zhangqinglong.zql/assets/"
```

Comment:
local DIR should be removed first

Reply:
I intend to modify the string to fit the environments value.